components/src/dynamo/vllm/multimodal_handlers/worker_handler.py

```
@@ -0,0 +1,272 @@
+            # Create a descriptor based on the embedding shape.
+            embeddings = torch.empty(
+                request.embeddings_shape,
+                dtype=self.EMBEDDINGS_DTYPE,
+                device=self.EMBEDDINGS_DEVICE,
+            )
+            descriptor = connect.Descriptor(embeddings)
+
+            if descriptor is None:
+                raise RuntimeError(
+                    "Descriptor is None in PD worker - cannot process embeddings"
+                )
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)